dev/constraints.h

```diff
@@ -481,7 +481,6 @@ namespace sqlite_orm {
                                                              check_if_is_type<null_t>,
                                                              check_if_is_type<not_null_t>,
                                                              check_if_is_type<unindexed_t>,
-                                                             check_if_is_template<prefix_t>,
```

Comment:
are you sure about that? the template is named `is_constraint` not `is_column_constraint`. There are two types of constraints: table constraints and column constraints. E.g. FOREIGN KEY is table constraint and it exists in this template. So I doubt we need to remove `prefix_t` from this list

Reply:
I had in mind that `is_constraint` is only ever used to check column constraints in `make_column()`.
However, your point is fair enough. See next commit.